src/entry.cpp

```diff
@@ -637,6 +637,7 @@ namespace {
 	std::string entry::to_string() const
 	{
 		std::string ret;
+		ret.reserve(256);
```

Comment:
If this is done at all it should definitely only be done for lists and dicts. This is effectively defeating the Short String Optimization for small entries.